substra/sdk/__init__.py

```diff
@@ -14,8 +14,10 @@
 
 from substra.sdk.client import Client
 from substra.sdk.utils import retry_on_exception
+from substra.sdk.backends.local.backend import DEBUG_OWNER
 
 __all__ = [
     'Client',
     'retry_on_exception',
+    'DEBUG_OWNER',
```

Comment:
To facilitate usage, could we also add it to substra/__init__.py so that users can write `substra.DEBUG_OWNER` instead of `substra.sdk.DEBUG_OWNER` ? The SDK is an internal structure that we don't want to expose to users.